src/main/java/io/quarkus/bot/AffectNeedsRebaseToOpenedPullRequests.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.event.Push;
+import io.quarkus.bot.util.Labels;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+
+import java.io.IOException;
+
+public class AffectNeedsRebaseToOpenedPullRequests {
+
+    void triageNeedsRebase(@Push GHEventPayload.Push pushRequestPayload) throws IOException {
+        GHRepository repository = pushRequestPayload.getRepository();
+        for (GHPullRequest pullRequest : repository.getPullRequests(GHIssueState.OPEN)) {
+            Boolean mergeable = pullRequest.getMergeable();
+            if (mergeable != null) {
+                if (mergeable) {
+                    pullRequest.removeLabels(Labels.TRIAGE_NEEDS_REBASE);
```

Comment:
Another thing: I think you would need to take into account when a PR is pushed. As there's a good chance the rebase status will need to be cleaned up.